gettsim/taxes/zu_verst_eink/zu_verst_eink.py

```diff
@@ -15,48 +15,67 @@
 """
 
 
-def freibeträge(
+def individual_freibeträge(
```

Comment:
```suggestion
def freibeträge_ind(
```
I think we should stick with the convention that the unit of aggregation is appended as suffix, even if it is more a demarcation from other stuff here.

Reply:
You're right! I will change it.